### ISSUE TITLE
Create a settings panel for codegen defaults

### ISSUE THREAD

**Issue by jimschubert** [jimschubert/intellij-swagger-codegen#3]
This should be per-project settings

Probably a good place for defaults like default config file, file encoding, etc.

**Comment by NabiKAZ** [comment]
Yeah, each when I want use it, must be entry output directory path and set self settings. It's time-consuming and repetitive.

**Comment by jimschubert** [comment]
I hope to have Swagger Codegen dumping generator options to a hidden directory in the next week or two. Then, once released, I will need to consume that file for this issue.

Another related issue is to support Swagger Codegen external configs.

I've been moving somewhat slowly on open source lately, because I have an 8 month old baby at home. If it's anything like with my first kid, I'll start having more free time in the next couple of months.

**Comment by NabiKAZ** [comment]
@jimschubert 
Congratulations on the new arrival to your family :-D
